tests/e2e/workloads/ocp/logging/test_openshift-logging.py

```diff
@@ -27,7 +27,7 @@
 logger = logging.getLogger(__name__)
 
 
-@skipif_ocs_version("4.12")
+@skipif_ocs_version("<4.12")
```

Comment:
It should be skip for only 4.12 so it should be "==4.12"

Reply:
done